dredd/api-description.yml

```diff
@@ -994,6 +994,33 @@ paths:
         400:
           $ref: '#/responses/error'
           x-disabled: true
+  /history:
+    get:
+      summary: Return history entries to a specific show
+      description: |
+        The history endpoint returns logged activities stored in the history table, like episodes snatched and downloaded. Or downloaded subtitles for an episode.
+      parameters:
+        - $ref: '#/parameters/page'
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/sort'
+      responses:
+        200:
+          $ref: '#/responses/pagination_history'
+          x-disabled: true
+        400:
+          $ref: '#/responses/error'
+          description: Invalid series id or pagination parameters
+          x-request:
+            path-params:
+              seriesid: asdf
+          x-disabled: true
+        404:
+          $ref: '#/responses/error'
+          description: Series not found
+          x-request:
+            path-params:
+              seriesid: tvdb999999999
+          x-disabled: true
```

Comment:
@medariox @sharkykh yes i'm disabling these.
I'm not getting it to work. If some of you want to spend time on this, feel free.